Warn about sourcemap leakage (#10637)
getsentry/sentry-docs: platform-includes/sourcemaps/overview/javascript.angular.mdx

}
```

<Alert>

>   Generating sourcemaps may expose them to the public, potentially causing your
>   source code to be leaked. You can prevent your source maps from being publicly
>   accessible by configuring your server to deny access to `.js.map` files, or by
>   deleting the sourcemaps before deploying your application.

Reviewer: ```suggestion
Generating sourcemaps may expose them to the public, potentially causing your source code to be leaked. You can prevent this by configuring your server to deny access to `.js.map` files, or by deleting the sourcemaps before deploying your application.
```
